.htaccess

```diff
@@ -22,11 +22,17 @@ RewriteRule .* - [E=HTTP_AUTHORIZATION:%{HTTP:Authorization},L]
 </IfModule>
 
 <IfModule mod_alias.c>
-    # Block access to VCS directories
+    # Restrict access to VCS directories
     RedirectMatch 404 /\\.(svn|git|hg|bzr|cvs)(/|$)
 
     # Restrict access to root folder files
-    RedirectMatch 404 /(composer\.(json|lock)|README\.md|UPGRADE\.md)$
+    RedirectMatch 404 /(autoload\.php|composer\.(json|lock|phar)|README\.md|UPGRADE-(.*)\.md|CONTRIBUTING\.md|eula.*\.txt|\.gitignore|.*\.dist|\.env.*)$
+
+    # Restrict access to shop configs files
+    RedirectMatch 404 /(web\/cache\/(config_\d+\.json|all.less|))$
```

Comment:
did you miss the .env files like here? https://github.com/shopware/shopware/pull/1499/files#diff-8052c42ab3b8aa06a3f5f788a4ddccc2R32